client/src/pages/OrganizationProfile.js

```diff
@@ -193,12 +193,12 @@ const OrganizationProfile = () => {
       const offerHelp = Object.values(objectives).some((val) => val === true);
       const { address } = location;
       const nameArr = name.split(" ");
-      if (nameArr.length < 2) {
+      if (nameArr.length < 1) {
         firstName = nameArr[0];
         lastName = firstName.split("").pop();
       } else {
         firstName = nameArr[0];
-        lastName = nameArr[1];
+        lastName = nameArr[0];
```

Comment:
I mean if you have an array with the whole name, i.e `["Omar", "Zoghayyer"]` then this will give out initials such as `OO` because you are setting the last name to the first element of the array, assuming that it is an array with the full name broken down in parts.